client/app/api/course/Assessment/Assessments.js

```diff
@@ -35,6 +35,19 @@ export default class AssessmentsAPI extends BaseCourseAPI {
     return this.client.get(`${this.#urlPrefix}/${assessmentId}/edit`);
   }
 
+  liveFeedbackSettings(assessmentId) {
+    return this.client.get(
+      `${this.#urlPrefix}/${assessmentId}/live_feedback_settings`,
+    );
+  }
+
+  updateLiveFeedbackSettings(assessmentId, params) {
+    return this.client.patch(
+      `${this.#urlPrefix}/${assessmentId}/update_live_feedback_settings`,
```

Comment:
Update this route to follow updated `routes.rb`

Reply:
done